### ISSUE TITLE
[BUG] Adding Soundhound ID Now Widget Fails

### ISSUE THREAD

**Issue by themarkwilliams** [Deletescape-Media/Lawnchair#4277]
### Describe the bug

Great launcher. I'm trying to add the **SoundHound' ID Now** widget to a couple locations with lawnchair. 

The widget is not added when dropping it.

### Steps to reproduce

1. With SoundHound app installed, hold home screen
2. Choose Widgets
3. Find and Expand ShoundHound widgets
4. Hold SoundHound ID Now
5. Attempt to drop on any screen
6. Drop action fails and widget is not added.

### Expected behavior

Widget added to selected location

### Device information

- Device: Pixel 7a
- OS: Android 14 with March 1, 2024 updates. Security updates: March 5, 2024
- App version: version 14 Beta 2

### Additional context

I don't think this is causing a lawnchair crash, but I tried the steps described in the FAQ and that didn't work either. 

I removed the ID Now widgets from the Pixel Launcher and that had no impact.

**Comment by joelbu** [comment]
I can reproduce the same behaviour on a Pixel 6 with Android 14 (Build AP2A.240705.004) and Lawnchair 14 beta 2. It didn't matter wether Lawnchair or Nova was my default homescreen, the bug happened regardless.

**Comment by Feliksas** [comment]
Still reproducible with Pixel 9 Pro XL

**Comment by MrSluffy** [comment]
Can you try and check latest nightly if still persists reopen this issue

**Comment by joelbu** [comment]
I just tried with Version Version 15.Dev.(#1780) app.lawnchair.debug on Android 14 (AP2A.240905.003.F1) and I see the same behaviour. I can drop the soundhound 4x1 widget but not the 1x1. I can also drop other 1x1 widgets normally.

Edit: Simply replying didn't reopen and I can't see how to manually reopen, maybe I don't have the rights.

**Comment by MrSluffy** [comment]
That's odd it's working fine here
![Screenshot_20241215-155250_Lawnchair (Debug)](https://github.com/user-attachments/assets/d4317681-7149-4bd3-b610-593cf91abc08)

**Comment by MrSluffy** [comment]
Did you set lawnchair as Default home
![Screenshot_20241215-155505_Lawnchair (Debug)](https://github.com/user-attachments/assets/77110c76-0225-4472-8782-92421ac75f84)
![Screenshot_20241215-155519_Lawnchair (Debug)](https://github.com/user-attachments/assets/39e3b20b-88d4-4250-b793-cbb981205f61)

**Comment by MrSluffy** [comment]
for 1x1 looks like a bug also tried in pixel launcher it doesn't work too

**Comment by joelbu** [comment]
Regarding for your question first: 14 beta 2 was my default, making the 15 nightly default doesn't change anything though.

Wow yeah you're right it doesn't drop in the Pixel Launcher. For me as well.

I came from Nova Launcher before Lawnchair, and weirdly enough it works in Nova Launcher, even now, I just reinstalled it to test.

Does Lawnchair derive from the AOSP Launcher? Maybe it's some shared code between Lawnchair and Pixel launcher that triggers the behaviour. Might well be a Soundhound bug though.
